sqlalchemy_jsonapi/unittests/test_serializer_get_collection.py

```diff
@@ -106,7 +106,7 @@ def test_get_collection_response_with_single_include_model(self):
                     }
                 },
                 'attributes': {
-                    'content': 'This is a comment'
+                    'content': u'This is a comment'
```

Comment:
Because this string is an actual attribute that is pulled from the database, whereas the others are not.